PowerUp/app/src/main/java/powerup/systers/com/MainActivity.java

```diff
@@ -9,34 +9,38 @@
 import android.widget.ImageButton;
 
 public class MainActivity extends Activity {
+    private boolean previouslyStarted;
 
-	@Override
-	protected void onCreate(Bundle savedInstanceState) {
-		super.onCreate(savedInstanceState);
-		setContentView(R.layout.activity_main);
-		SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(getBaseContext());
-		boolean previouslyStarted = prefs.getBoolean(getString(R.string.pref_previously_started), false);
-		if (!previouslyStarted) {
-			startActivity(new Intent(MainActivity.this, AvatarRoom.class));
-		}
-		ImageButton new_user = (ImageButton) findViewById(R.id.newUserButtonFirstPage);
-		new_user.setOnClickListener(new View.OnClickListener() {
-			@Override
-			public void onClick(View v) {
-				Intent myIntent = new Intent(MainActivity.this,
-						AvatarRoom.class);
-				startActivityForResult(myIntent, 0);
-			}
-		});
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(getBaseContext());
+        previouslyStarted = prefs.getBoolean(getString(R.string.pref_previously_started), false);
 
-		ImageButton start = (ImageButton) findViewById(R.id.startButtonMain);
-		start.setOnClickListener(new View.OnClickListener() {
-			@Override
-			public void onClick(View v) {
-				Intent myIntent = new Intent(MainActivity.this, MapActivity.class);
-				startActivityForResult(myIntent, 0);
-			}
-		});
+        ImageButton new_user = (ImageButton) findViewById(R.id.newUserButtonFirstPage);
+        new_user.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Intent myIntent = new Intent(MainActivity.this,
+                        AvatarRoom.class);
+                startActivityForResult(myIntent, 0);
+            }
+        });
 
-	}
+        ImageButton start = (ImageButton) findViewById(R.id.startButtonMain);
+        start.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if (!previouslyStarted) {
+                    startActivity(new Intent(MainActivity.this, AvatarRoom.class));
+                    previouslyStarted = true;
```

Comment:
The previouslyStarted variable has been temporarily set to true here so that pressing start without closing application does not lead user back to change room.
The shared preference value is set to true only after AvatarActivity as in case the avatar is not chosen and application is closed, change room should open on pressing start the next time.